tour/ranges-splits.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Ranges and Splits
+---
+
+A Range is a specified group of Keys. There are many different ways to create a Range.  Here are a few examples:
+```java
+new Range(Key startKey, Key endKey)  // Creates a range from startKey inclusive to endKey inclusive.
+new Range(CharSequence row)  // Creates a range that covers an entire row.
+new Range(CharSequence startRow, CharSequence endRow) // Creates a range from startRow inclusive to endRow inclusive.
+```
+
+A Scanner by default will scan all Keys in a table but this can be inefficient. It is a good practice to set a range on a Scanner.
+```java
+scanner.setRange(new Range("id0000", "id0010"));  // returns rows from id0000 to id0010
+```
+
+As your data grows larger, Accumulo will split tables across multiple servers called Tablet Servers.  
+By default a table will get split on row boundaries, guaranteeing an entire row to be on one Tablet Server.  We have the ability to 
+tell Accumulo were to split tables by setting split points. This is done using _addSplits_ in the [TableOperations] API.  The image below 
+demonstrates how Accumulo splits data.  
+
+![data distribution]({{ site.url }}/images/docs/data_distribution.png)
+
+There are many useful administrative methods in [TableOperations] so take a minute to look through the API. Here are some terms specific to Accumulo:
```

Comment:
This is good content, but it feels out of place here.

Reply:
I agree. I was trying to point out the more useful API classes and the best places to start.  Maybe we could do that in another exercise